nette/notejam/src/Notejam/Notes/Note.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace Notejam\Notes;
+
+use Doctrine\ORM\Mapping as ORM;
+use Kdyby\Doctrine\Entities\Attributes\Identifier;
+use Kdyby\Doctrine\Entities\MagicAccessors;
+use Notejam\Pads\Pad;
+use Notejam\Users\User;
+
+
+
+/**
+ * @ORM\Table(name="notes")
+ * @ORM\Entity(repositoryClass="Notejam\Notes\NoteRepository")
+ * @ORM\HasLifecycleCallbacks()
+ */
+class Note
+{
+
+    use Identifier;
+    use MagicAccessors;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="name", type="string", length=100)
+     */
+    private $name;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="text", type="text")
+     */
+    private $text;
+
+    /**
+     * @var User
+     *
+     * @ORM\ManyToOne(targetEntity="Notejam\Users\User", inversedBy="notes")
+     * @ORM\JoinColumn(nullable=FALSE)
+     */
+    private $user;
+
+    /**
+     * @var Pad
+     *
+     * @ORM\ManyToOne(targetEntity="Notejam\Pads\Pad", inversedBy="notes")
+     */
+    private $pad;
+
+    /**
+     * @var \Datetime
+     *
+     * @ORM\Column(name="updated_at", type="datetime")
+     */
+    private $updatedAt;
+
+
+
+    public function __construct(User $user)
```

Comment:
Not really, since the identity is the entity. And the relation is for the entity, not for the relation.